ble/services/EddystoneService.h

```diff
@@ -185,6 +185,29 @@ class EddystoneService
         return false;
     }
 
+    /**
+     *  Set Eddystone URL Frame information.
+     *  @param[in] power          TX Power in dB measured at 0 meters from the device.
+     *  @param[in] url            Encoded URL
+     *  @param[in] urlAdvPeriodIn How long to advertise the URL frame (measured in # of adv periods)
+     *  @return false on success, true on failure.
```

Comment:
The reason why I left it "false on success, true on failure" is because I used the existing function "bool setURLFrameData(int8_t power, const char *urlIn, uint32_t urlAdvPeriodIn)" as a template. So the return values remain unchanged.